src/entrypoints/popup/app.css

```
@@ -61,10 +61,32 @@ h1 {
     margin-top: 2rem;
 }
 
+.reset-container {
+    width: 306px;
+    text-align: left;
+    margin-top: -18px;
+}
+
+.reset-container a {
+    font-size: 0.75rem;
+    font-family: "Manrope", sans-serif;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Manrope" (font-family-name-quotes)

(font-family-name-quotes)